### DISCUSSION TITLE
Check out cool esmeta project

### DISCUSSION THREAD

**Discussion by safinaskar** [kaleidawave/ezno#178] [Ideas]
https://github.com/es-meta/esmeta . This is Scala program, which parses ecmascript spec and... converts it into actual interpreter!

**Comment by kaleidawave** [comment]
Cool, haven't heard of it before. That is a neat idea! At this stage a specification => code program isn't really a possibility for solving the goals of this type checking project. Thanks for sharing anyway 👍
